nixos/lib/testing.nix

```diff
@@ -74,6 +74,7 @@ rec {
     , makeCoverageReport ? false
     , enableOCR ? false
     , name ? "unnamed"
+    , extraQemuOpts ? null # Extra options for QEMU passed to all VM's. String expected
```

Comment:
Would a list works here/make sense? There would be also escapeShellArgs (with an trailing `s`).